### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by 0xflotus** [Remix-Design/RemixIcon#480] [Ideas]
Is there a dark theme for the website? If not, please consider to add one :)

<img width="1669" alt="Bildschirmfoto 2023-01-08 um 22 34 52" src="https://user-images.githubusercontent.com/26602940/211220271-28ad3787-3ad6-478e-801f-337cee6d253e.png">

**Comment by xiaochunjimmy** [comment]
We added the dark theme for the website, try it out!
https://remixicon.com/
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/6701827/229299270-c0868c28-4f32-44b9-97d8-eb09522c8844.png">
